src/d3a/d3a_core/global_objects.py

```diff
@@ -15,29 +15,39 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+from d3a.models.strategy.external_strategies import ExternalMixin
+from d3a.d3a_core.util import get_market_maker_rate_from_config
 
 
-class GlobalObjects:
+class GlobalStatistics:
 
-    def __init__(self):
-        self.area_tree_dict = {}
+    def __init__(self, root_area):
+        self.area_stats_tree_dict = {}
+        self.root_area = root_area
 
-    def update(self, area):
-        if area.current_market:
-            self._create_grid_tree_dict(area, self.area_tree_dict)
+    def update(self):
+        if self.root_area.current_market:
+            self._create_grid_tree_dict(self.root_area, self.area_stats_tree_dict)
 
     def _create_grid_tree_dict(self, area, outdict):
-        outdict[area.name] = {}
+        outdict[area.uuid] = {}
         if area.children:
             if area.current_market:
-                area_dict = {"last_market_slot": area.current_market.time_slot_str,
-                             "last_market_bill": area.stats.get_last_market_stats_for_grid_tree(),
-                             "last_market_stats": area.stats.get_price_stats_current_market(),
-                             "last_market_fee": area.current_market.fee_class.grid_fee_rate,
-                             "next_market_fee": area.get_grid_fee(),
-                             "children": {}}
+                area_dict = {'market_maker_rate': get_market_maker_rate_from_config(
+                                                  area.current_market),
+                             'last_market_bill': area.stats.get_last_market_stats_for_grid_tree(),
+                             'last_market_stats': area.stats.get_price_stats_current_market(),
```

Comment:
Would be a good opportunity to settle on terms `last_market` vs `current_market`

Reply:
Please make a task for research/testing team to settle on nomenclature when to use `last_market/current_market/next_market`

Reply:
If the time is 10:13, energy is being delivered for the 10:00 market slot, and the current spot market is the range 10:15-10:30. The active spot market is considered 'current'.

last_market = 10:00-10:15
current_market = 10:15-10:30
next_market = 10:30-10:45